Makefile

```diff
@@ -41,12 +41,18 @@ tre-deploy:
 	&& . ./devops/scripts/load_env.sh ./templates/core/terraform/.env \
 	&& cd ./templates/core/terraform/ && ./deploy.sh 
 
+letsencrypt:
```

Comment:
```suggestion
.PHONY: letsencrypt
letsencrypt:
```

Reply:
Merged with `develop`, updated [docs/quickstart.md](https://github.com/microsoft/AzureTRE/blob/develop/docs/quickstart.md), and added `letsencrypt` to the `.PHONY` list at the start of the __makefile__.